### DISCUSSION TITLE
Support for centralized deployer usage

### DISCUSSION THREAD

**Discussion by marvinhinz** [deployphp/deployer#2469] [General]
Hi,

for various reasons we store our deploy scripts centrally on our ssh jump server, that every developer has an account on. 
Right now we use a little bash wrapper script, which works like this: `dep-wrap project action param1 param2` etc.
All deploy.php-files are stored in a projects subfolder. 

This enables us to deploy these projects without having to specify the `--file=.../projects/$project/deploy.php` param every time. 

I thought it would be nice to have this functionality native in deployer. Bash completion for available projects would be 
cool too! I'd like to create a pull request, when i have some spare time, but first i'm interested in feedback about this idea. 
What do you think?

**Comment by antonmedv** [comment]
This sounds cool, but looks like it’s really specific to your project.
